apiary.apib

```diff
@@ -1661,9 +1661,261 @@ root will not be listable, as it is virtual. Access known paths directly.
 
 # Group name
 
-## publish
+IPFS namespace (IPNS) tool
```

Comment:
Not sure about that. I'd rather not add links which aren't part of the CLI mans.